docs/source/userguide/creating_a_new_design.rst

```diff
@@ -0,0 +1,163 @@
+
+If a Block is `removed <removing_a_block_from_a_design_>` from a Design it is immediately available again for selection in the Block Palette.
+
+
+Specifying Block Attributes
+---------------------------
+
+The behaviour of a `block_` is defined via its `attributes <attribute_>`.  Attributes are pre-defined based on the function of the Block and may include default values providing a starting point for later implementation-time customisation.  A full list of the attributes associated with each Block available from the Block Palette can be found in the `PandABlocks-FPGA`_ documentation.
+
+Types of Attributes
+^^^^^^^^^^^^^^^^^^^
+
+`block_` attributes come in two forms:
+
+    #. An `immutable_attribute_` is one that once set within the `Design` cannot be modified by any action within the system.  It holds a constant value until manually modified as part of a controlled Design change.
+    #. A `writeable_attribute_` is one whose value can be dynamically modified via a 'put' request executed in response to an action within a pre-cursor Block's execution.
+
+Attributes whose value can be specified at design-time are denoted by a highlight below the attribute value field.
+
+Manually Setting or Modifying a Block Attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Block attributes are specified via the 'Block Information Panel' associated with the Block you wish to configure.
+
+To configure an attribute:
+
+    #. Select the Block you wish to configure by clicking on it within the Layout Panel.  The selected Block will be highlighted and the Block Information Panel associated with it displayed on the right-hand side of the user interface.
+    #. Find the Attribute you wish to configure in the list of available Attributes.
+    #. Edit the Attribute value field as necessary:
+
+        * If the Attribute represents a list of pre-defined options select your desired value from the drop-down list.  The Attribute value field updates to reflect the selected value.
+        * If the Attribute represents an enable/disable option select the checkbox to enable or disable the Attribute.  If the checkbox is empty the Attribute is disabled .  When enabled a tick is displayed within the checkbox.  
+        * If the Attribute requires a manually specified value select the Attribute value field by clicking within it.  Delete any pre-existing content and enter your desired value.  Press the *enter* key for the value to be submitted and recorded.  Values that have been edited but not yet submitted are denoted with a 'pencil' icon.  Upon successful submission the pencil is replaced by the default information symbol.
+
+         **NB:** No data type validation is performed on manually entered values.
+
+During the registration process of a new Attribute value a spinning icon is displayed to the left of the modified Attribute.  For more information on the process this represents see `attribute_change_lifecycle_`.
```

Comment:
Put rather than registration process

Reply:
'Registration process' is the wrong term here but within this context 'put' is also not quite what I was  aiming for. I've changed 'registration' for 'save', and later where we introduce the attribute change lifecycle I've added a description of the 'put' process as a combination of saving the attribute and applying it to the design.

Reply:
I'd prefer to avoid "Save" as it conflicts with using the "Save" button to save a design. 

How about "While the value is being sent to the server"?